doc/ext/doctest.rst

```diff
@@ -73,6 +73,19 @@ a comma-separated list of group names.
      explicit flags per example, with doctest comments, but they will show up in
      other builders too.)
 
+   * ``pyversion``, a string option, can be used to specify the required Python
+     version for the example to be tested. For instance, in the following case
+     the example will be tested only for Python versions greather than 3.3::
+
+         .. doctest::
+            :pyversion: > 3.3
+ 
```

Comment:
I am sorry @tk0miya , I checked Travis but I was not able to figure out where the problem was, and I thought to fix it on this morning. I have not enough confidence with all the tools yet :-/